core/src/main/java/org/testcontainers/containers/image/pull/policy/PullPolicy.java

```diff
@@ -0,0 +1,38 @@
+package org.testcontainers.containers.image.pull.policy;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Convenience class with logic for building common {@link ImagePullPolicy} instances.
+ *
+ */
+public class PullPolicy {
+
+    /**
+     * Convenience method for returning the default image pull policy, which pulls the image only if it does not exist
+     * @return {@link ImagePullPolicy}
+     */
+    public static ImagePullPolicy Default() {
```

Comment:
Silly style thing, but please could these method names start with a lower case character?

I was tempted to ask why this wouldn't be an enum, but can probably foresee users wanting to use more than these three, so it's OK to have these static factory methods.